jwst/scripts/pointing_summary.py

```diff
@@ -1,48 +1,57 @@
 #!/usr/bin/env python
 
-"""Pointing verification
+"""
+Pointing verification.
 
 Review contents of a set of given models for pointing information.
```

Comment:
For the other scripts, it looks like you moved the extended description into `main()`.  Would it make sense to do the same thing here?

FWIW I'm not sure what the "best practice" is.  If you put this into "main", is the extended description accessible via `pointing_summary --help`?

As a more general question, I don't see where these scripts are documented at https://jwst-pipeline.readthedocs.io/en/latest/jwst/package_index.html.  Do you happen to know where, if anywhere, the docs exist in our readthedocs build?

Reply:
yep, the idea is that the description shows up with --help. 

I only found it here: https://jwst-pipeline.readthedocs.io/en/latest/jwst/lib/set_telescope_pointing.html

Reply:
Depending on what INS says regards usage of this particular script, we can add more documentation later